src/BackupMetadata.cs

```diff
@@ -39,12 +41,17 @@ public int GetHashCode(BackupMetadata obj)
   /// </summary>
   public class BackupMetadata : ICloneable
   {
+
+    public BackupMetadata()
+    {
+      PhysicalDeviceNames = new List<string>();
+    }
     public decimal CheckpointLsn { get; set; }
     public decimal DatabaseBackupLsn { get; set; }
     public string DatabaseName { get; set; }
     public decimal FirstLsn { get; set; }
     public decimal LastLsn { get; set; }
-    public string PhysicalDeviceName { get; set; }
+    public List<string> PhysicalDeviceNames { get; set; }

```

Comment:
I see where you're going with this... I prefer you not change this to a `List`.  This class is meant to represent 1 return row of data from `msdb.dbo.backupset`.
If the goal is to bundle the striped backups together, I suggest you create a new data type joins striped records into a list where needed

Reply:
Yeah...I was debating on which approach to use as well

I feel `BackupMetadata` represents an instance of a backup, not a row of data from the query. That's why I changed the backupfile field from a single string to a list.
I can see previously, that it was synonymous with a row of data returned from the query too.

I'd like to argue, this is still a better approach than creating a new class to represent Striped backups.

However, if we do need to - most of the properties and behaviors will be the same.
So I'm thinking...it might be better to create it as a subclass of `BackupMetadata`?